src/commands/configure/group.rb

```diff
@@ -24,29 +47,12 @@ def answers_file
           config.group_answers_file(group_name)
         end
 
-        attr_reader :group_name
-
         def higher_level_answer_files
           [config.domain_answers_file]
         end
 
         def record_primary_group
-          unless primary_group_recorded?
-            primary_groups << group_name
-            Data.dump(Constants::GROUPS_CACHE_PATH, groups_cache)
-          end
-        end
-
-        def primary_group_recorded?
-          primary_groups.include? group_name
-        end
-
-        def primary_groups
-          groups_cache[:primary_groups] ||= []
-        end
-
-        def groups_cache
-          @groups_cache ||= Data.load(Constants::GROUPS_CACHE_PATH)
+          cache.add(group_name) unless cache.group?(group_name)
```

Comment:
Rather than asking the cache if it already contains the group, and then adding it if it doesn't, can we just tell it to add the group and let it decide if it should record it? I.e.: can we just do `cache.add` here and have that contain the conditional? I think this is a situation where it would be useful to follow [tell don't ask](https://robots.thoughtbot.com/tell-dont-ask).